R/sensitivity-spider-plot.R

```diff
@@ -270,7 +290,7 @@ sensitivitySpiderPlot <- function(sensitivityCalculation,
 
       plot <- ggplot(
         dataSubset,
-        aes(x = ParameterFactor, y = !!sym(yColumn), group = ParameterPath)
+        aes(x = !!sym(xColumn), y = !!sym(yColumn), group = ParameterPath)
```

Comment:
I would not use bang bang and sym syntax here. Please use the recommended `.data` pronoun (cf [here](https://ggplot2.tidyverse.org/articles/ggplot2-in-packages.html#using-aes-and-vars-in-a-package-function)).
I would take the opportunity to update the rest of the code.

Reply:
Thanks, it's more explicit that way